content/en/docs/concepts/services-networking/_index.md

```diff
@@ -7,44 +7,214 @@ description: >
+  
+* L2bridge is a virtual L2 bridge that allows attachment, configuration and communications between pods on the same node. Depending on your deployment of container runtimes and CNI plugins, the bridge function could be implemented within the CNI plugin itself.  
+
+* [Network plugins](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/) allow pods to communicate even when the source pod destination pod are running on different nodes. Network plugins achieve this in different ways. Networks plugins are also known as _CNI plugins_.
+  
+* Virtual overlay network consists of a network of tunnels connecting pods running on different nodes.
+
+* Physical underlay network, composed of L2 switches, L3 routers, load balancers for example, transport packets between nodes. 
+
+Kubernetes networking supports container network connectivity over different cluster topologies. You can look over some of the container networking functions described in the _examples_ below.
+
+* [Containers on the same pod](#containers-on-the-same-pod)
+* [Pods on the same node](#pods-on-the-same-node)
+* [Pods on different nodes using a virtual overlay network](#pods-on-different-nodes-using-a-virtual-overlay-network)
+* [Pods on different nodes using a physical underlay network](#pods-on-different-nodes-using-a-physical-underlay-network)
+
+
+You might discover a different variant of this architecture and components that you prefer to run in your environment. However, it should meet some or all of the network requirements noted above.
+
+### Containers on the same pod
+
+Containers within a pod share their own network namespaces including their IP address and MAC addresses.
+
+Containers within a pod can all reach each other's ports on
+`localhost:port number`. If you examine a pod you will see a `lo0` interface used for localhost communications between containers in the pod network namespace. 
+
+Figure 2 illustrates an example of localhost communications between two containers on the same pod. You define a pod 1 network namespace that provides pod 1 with its own isolated network stack. Your containers on pod 1 share a single IP and MAC address; Containers on pod 1 are configured with separate port numbers.
+
+{{< figure src="/docs/images/k8net-localhost.drawio.svg" alt="k8s localhost" class="diagram-large" caption="Figure 2. Example of container communications  within the same pod using localhost:port number" >}}
+
+Your containers within a pod must coordinate port usage, but this is no different from processes in a VM.  
+
+How you implement this is a detail of the particular container runtime you employ in your environment.
+
+It is possible to request ports or call host ports on the node itself which forward packets to your pod. This is a very niche operation. How you implement this is a detail of the container runtime. The pod itself is blind to the existence or non-existence of host ports.
```

Comment:
I would not mention that detail here; I'm not even sure how you configure it within Kubernetes.